cypress/composables/webhooks.ts

```diff
@@ -0,0 +1,81 @@
+import { BACKEND_BASE_URL } from '../constants';
+import { NDV, WorkflowPage } from '../pages';
+import { getVisibleSelect } from '../utils';
+
+export const waitForWebhook = 500;
+
+export interface SimpleWebhookCallOptions {
+	method: string;
+	webhookPath: string;
+	responseCode?: number;
+	respondWith?: string;
+	executeNow?: boolean;
+	responseData?: string;
+	authentication?: string;
+}
+
+const workflowPage = new WorkflowPage();
+const ndv = new NDV();
+
+export const simpleWebhookCall = (options: SimpleWebhookCallOptions) => {
```

Comment:
Extracted out of webhooks tests. The tests were running multiple times due to importing.